### PR TITLE
Added vertex delta k8800

### PR DESCRIPTION
**Pull request by velbn**
We've added the config for the K8800 machine

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Developers, refer to issue CURA-4312.

**Comment by alekseisasin** [comment]
velbn, Could you please make few changes in vertex_delta_k8800.def.json file:
-> The version should be 2;
-> "inherits": "fdmprinter", Not fdmprinter.json
-> you override settings, then define them within "overrides": {    ....    }
-> do you want to add some information for "metadata" : { }

**Comment by velbn** [comment]
Hi I've updated the definition file

File: resources/definitions/vertex_delta_k8800.def.json [hunk 1]
```diff
@@ -1,65 +1,62 @@
...
+      "default_value": 0
+    },
+    "machine_nozzle_offset_x_1": {
+      "default_value": 0
+    },
+    "machine_nozzle_offset_y_1": {
+      "default_value": 0
+    },
+    "machine_gcode_flavor": {
+      "default_value": "RepRap"
```
Comment (LipuFei) [inline resources/definitions/vertex_delta_k8800.def.json]: This value should be one of the following:
- `"RepRap (Marlin/Sprinter)"`
- `"RepRap (Volumetric)"`
- `"RepRap (RepRap)"`

File: resources/definitions/vertex_delta_k8800.def.json [hunk 2]
```diff
@@ -1,65 +1,62 @@
...
-        "machine_end_gcode": {
-            "default": "; Vertex Delta end code\nM107 ; Turn off fan\nG91 ; Relative positioning\nT0\nG1 E-1 F1500; Reduce filament pressure\nM104 T0 S0\nG90 ; Absolute positioning\nG92 E0 ; Reset extruder position\nM300 S4000 P500\nM300 S3000 P500\nM300 S2000 P800\nG28\nM84 ; Turn steppers off"
-        }
+  "id": "K8800",
+  "name": "Vertex Delta K8800",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "manufacturer": "Velleman nv",
+	"file_formats": "text/x-gcode",
```
Comment (LipuFei) [inline resources/definitions/vertex_delta_k8800.def.json]: Please replace this `TAB` with `spaces`.

**Comment by velbn** [comment]
Fixed the things =)

**Comment by velbn** [comment]
One quick side question, when will this config be available? In the next release or when the user opens cura?

**Comment by LipuFei** [comment]
Now that this is merged, it will be in the next release :)

**Pull request merged**